app/components/results-list/component.js

```diff
@@ -47,30 +47,18 @@ export default Ember.Component.extend({
         },
 
         pageback() {
-            let queryParams = {};
             let page = Number(this.parameters["page"]);
-            if (!page) {
-                page = 0;
-            }
-            page = page-10;
-            if (page < 0) {
-                page = 0;
+            if (!page || --page < 1) {
```

Comment:
I think the comparison `--page < 1` is confusing to reason about...can you change it to `page <= 1`?